internal/vxproto/module.go

```diff
@@ -355,7 +368,10 @@ func (ms *moduleSocket) parseFileStream(ctx context.Context, packet *Packet) (bo
 	}
 
 	tempDir := filepath.Join(os.TempDir(), "vx-store")
-	os.Mkdir(tempDir, 0700)
+	err := os.Mkdir(tempDir, 0700)
```

Comment:
This is a good example how linter message is fixed.

Reply:
I didn't understand)))